universe/Planet.h

```diff
@@ -1,6 +1,7 @@
 #ifndef _Planet_h_
 #define _Planet_h_
 
+#include "Enums.h"
```

Comment:
> thoughts on extra include vs. initializer lists in .cpp file?

If it can be avoided (with `extern` eg) try to avoid it.  I would leave it as suggested in this case.